### PR TITLE
Update typography

### PR DESCRIPTION
**Pull request by tinykite**
## What this does 
Swaps in Neue Haas Grotesk for headings and body text throughout the LIL website. To avoid false italics and false bold being applied by browsers, this update additionally adjusts the font-weight for text site-wide, and uploads an italic style of  Neue Haas Grotesk. 

I noticed that the website was previously applying a bold weight site-wide and then removing it certain case-by-case instances. This update reverts all text to a roman/body weight, and applies a bold weight as needed. 

## Screenshots

Homepage hero 
Screenshot 2024-02-29 at 4 04 43 PM
Note: the line width of the introduction text, shown at large viewport sizes like what is pictured above, will be adjusted in a future update.

Homepage about section
Screenshot 2024-02-29 at 4 06 45 PM

Individual projects, from the projects page
Screenshot 2024-02-29 at 4 06 54 PM

Headshots, from the about page
Screenshot 2024-02-29 at 4 07 15 PM

Blog post
Screenshot 2024-03-04 at 9 40 47 AM

Italic body text in blog post
Screenshot 2024-03-04 at 9 41 10 AM

## How To Test
I've discovered that the least-error prone way to check out a branch locally is to not follow the instructions github provides on each PR. It feels much more consistent to:

- Add a new remote for my fork of website-static, named meaningfully (for example, named tinykite)
- Run `git pull tinykite` to make sure you have the latest remote branches
- Run `git checkout update-typography` to checkout this branch

### CODE REVIEW

File: app/_includes/about-people-affiliated.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       </div>
 
       {% if person.email or person.website or person.twitter or person.github %}
-        <ul>
+        <ul class="social">
```
Comment (tinykite) [inline app/_includes/about-people-affiliated.html:27]: Added this to apply the correct bold weight

File: app/assets/css/main.scss [hunk 2]
```diff
@@ -603,7 +628,7 @@ section, .container {
 
   li {
   line-height: 1.4;
-  // font-weight: 300;
+  // font-weight: 500;
```
Comment (tinykite) [inline app/assets/css/main.scss:631]: Updated this for consistency

**Review by rebeccacremona — APPROVED** [review]
Cool! Thanks for doing this.

> Note: the line width of the introduction text, shown at large viewport sizes like what is pictured above, will be adjusted in a future update.

Clare actually had me split that up into three lines for now: https://github.com/harvard-lil/website-static/pull/510

She reassured that is a temporary move and won't be part of the new design :-)

**Comment by tinykite** [comment]
> Cool! Thanks for doing this.
> 
> > Note: the line width of the introduction text, shown at large viewport sizes like what is pictured above, will be adjusted in a future update.
> 
> Clare actually had me split that up into three lines for now: #510
> 
> She reassured that is a temporary move and won't be part of the new design :-)

Ah, it sounds like there was more than one conversation happening about this! Jacob and I had previously had a conversation about this last Thursday — I was planning to fix it with this one-line CSS fix: 

```
.header-splash__description {
  max-width: 40ch;
}
```

**Comment by rebeccacremona** [comment]
Good to know! Moving the convo to Linear and `@`ing all involved parties. (Personally am a big fan of doing that CSS change instead.)

**Comment by tinykite** [comment]
I apologize for not creating an issue for it in Linear and perhaps causing duplicate conversations!

**Comment by rebeccacremona** [comment]
No big deal 😄

**Pull request merged**